### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by fede03billy**
Sample homepage

### CODE REVIEW

File: astro/src/components/Navbar.astro [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="flex justify-between items-center py-10 px-4">
+  <div class="container-1 flex items-center gap-5">
+    <div class="hamburger-menu"></div>
+    <div class="logo text-xl font-extrabold">BiciRubbate</div>
```
Comment (darioguarascio) [inline astro/src/components/Navbar.astro:4]: Bad! hardcoded strings versus `t()` syntax

File: astro/src/layouts/Base.astro [hunk 2]
```diff
@@ -28,46 +26,11 @@ const { title, description } = Astro.props;
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
     <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-    <!-- <meta name="generator" content={Astro.generator} /> -->
```
Comment (darioguarascio) [inline astro/src/layouts/Base.astro:31]: if you can leave commented these lines, will be used in the future

File: astro/src/models/listing.ts [hunk 3]
```diff
@@ -28,8 +28,9 @@ export default async function() {
 
     return (!data || data.length == 0) ? false : data;
 
-  } catch (e) {
-    throw e
```
Comment (darioguarascio) [inline astro/src/models/listing.ts:31]: leave the throwing to give errors, remove the console.log

File: astro/src/pages/index.astro [hunk 4]
```diff
@@ -2,22 +2,58 @@
...
+        90deg,
+        rgba(82, 241, 232, 0.24) 0%,
+        rgba(176, 253, 79, 0.24) 25%,
+        rgba(82, 241, 232, 0.24) 50%,
+        rgba(162, 251, 49, 0.24) 100%
+      );
+    }
+  </style>
+  <script>
+    document.getElementById("listing-button").addEventListener("click", () => {
```
Comment (darioguarascio) [inline astro/src/pages/index.astro]: use `<a href="/listing">`

File: astro/src/pages/index.astro [hunk 5]
```diff
@@ -2,22 +2,58 @@
...
+      window.location.href = "/listing";
+    });
+  </script>
+  <div class="w-full">
+    <div class="hero w-full h-[50vh] flex flex-col items-center justify-center">
+      <div class="pre-cta text-lg font-bold">
+        Discover the truth. Track down stolen bikes with our epic image search.
```
Comment (darioguarascio) [inline astro/src/pages/index.astro]: try using `{ t('pages.homepage.pre_cta') }`

**Review by darioguarascio — CHANGES_REQUESTED** [review]
Great OpenSourceSaturday

File: astro/src/pages/index.astro [hunk 6]
```diff
@@ -50,7 +45,7 @@ import BaseLayout from "@layouts/Base.astro";
       </div>
       <button
         class="list-button text-white bg-[#2DA771] rounded py-2 px-4"
-        id="listing-button">List Your Stolen Bike</button
+        id="listing-button"><a href="/listing">List Your Stolen Bike</a></button
```
Comment (darioguarascio) [inline astro/src/pages/index.astro]: a `a` tag inside a `button` is not a good practice.
you can have the `a` tag being displayed as a button by adding the same classes, and remove the `button` tag completely

**Review by darioguarascio — CHANGES_REQUESTED** [review]

File: astro/src/components/Navbar.astro (hunk 1 above)
Comment (fede03billy) [reply astro/src/components/Navbar.astro:4]: Is there some kind of logic to follow for naming `t()` function arguments that do not exists in the database yet?

e.g. `pages.homepage.cta`, `pages.homepage.description.bold` and so on nesting components

**Review by darioguarascio — APPROVED** [review]
Some strings needs to be converted in placeholders, but overall ok. we start having a shape!

File: astro/src/components/Navbar.astro (hunk 1 above)
Comment (darioguarascio) [reply astro/src/components/Navbar.astro:4]: @fede03billy the ideal would be `<page>`.`<component>`. `something`
and something like `_global`.`something` for shared . I dont know if there's a well established pattern for naming convention in this area, but this seems appropriate, especially keeping consistency between filenames, component names and string prefixes

**Pull request merged**